bot/exts/moderation/voice_gate.py

```diff
@@ -144,6 +166,10 @@ async def on_message(self, message: discord.Message) -> None:
 
         # When it's bot sent message, delete it after some time
         if message.author.bot:
+            # Comparing the message with the voice ping constant
+            if message.content.endswith(VOICE_PING[-45:]):
```

Comment:
What's the justification for `[-45:]`? I find this awkward. The message constant could simply not contain the mention placeholder. When you send the message, do:
```python
f"Hello, {member}! {MESSAGE}"
```
Then this holds with no slicing:
```python
message.content.endswith(MESSAGE)
```

Reply:
Hadn't thought about that.

Reply:
Changed in the latest commit.